server/src/main/java/com/cloud/hypervisor/kvm/discoverer/LibvirtServerDiscoverer.java

```diff
@@ -256,6 +257,11 @@ private void setupAgentSecurity(final Connection sshConnection, final String age
                 return null;
             }
 
+            if (SSHCmdHelper.sshExecuteCmd(sshConnection, "rpm -qa | grep -i ovmf", 3)) {
+                s_logger.debug("It's UEFI enabled KVM machine");
+                isUefiSupported = true;
+            }
+
```

Comment:
please extract, see above
I think the message should be: "This is a UEFI enabled KVM machine"